docs/user/alerting/view-alerts.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[view-alerts]]
+== View alerts
+:frontmatter-description: View and manage alerts in the {kib} {stack-manage-app} app.
+:frontmatter-tags-products: [kibana, alerting]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+When the conditions of a rule are met, it creates an alert.
+If the rule has actions, they run at the defined frequency.
+For example, the rule can send email notifications for each alert at a custom interval.
+For an introduction to the concepts of rules, alerts, and actions, refer to <<alerting-getting-started>>.
+
+You can manage the alerts for each rule in *{stack-manage-app}* > *{rules-ui}*.
+Alternatively, manage all your alerts in *{stack-manage-app}* > *Alerts*. preview:[] 
+
+[role="screenshot"]
+image::images/stack-management-alerts-page.png[Alerts page with multiple alerts]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+[NOTE]
+====
+You must have the appropriate {kib} {alert-features} and index privileges to view alerts.
+Refer to <<alerting-security,Alerting security requirements>>.
+====
+
+[discrete]
+[[filter-alerts]]
+=== Filter alerts
+
+preview::[]
+
+In *{stack-manage-app}* > *Alerts*, you can filter the list (for example, by alert status or rule type) and customize the filter controls.
+To search for specific alerts, use the KQL bar to create structured queries using {kibana-ref}/kuery-query.html[{kib} Query Language].
+
+By default, the list contains all the alerts that you have authority to view in the selected time period except those associated with Security rules.
+To view alerts for Security rules, click the query menu and select *Security rule types*:
+
+[role="screenshot"]
+image::images/stack-management-alerts-query-menu.png[The Alerts page with the query menu open]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+Alternatively, view those alerts in the {security-guide}/alerts-ui-manage.html[{security-app}].
+
+[discrete]
+[[view-alert-details]]
+=== View alert details
+
+To get more information about a specific alert, open its action menu (…) and select *View alert details* in either *{stack-manage-app} > Alerts* or *{rules-ui}*.
+There you'll see the current status of the alert, its duration, and when it was last updated.
+To help you determine what caused the alert, there is information such as the expected and actual threshold values and a summarized reason for the alert.
+
+If an alert is affected by a maintenance window, the alert details include its identifier.
+For more information about their impact on alert notifications, refer to <<maintenance-windows>>.
+
+[discrete]
+[[alert-status]]
+==== Alert statuses
+
+There are four common alert statuses:
+
+`active`:: The conditions for the rule are met and actions should be generated according to the notification settings.
+`flapping`:: The alert is switching repeatedly between active and recovered states.
```

Comment:
Just a tiny note on this: even if we represent `flapping` as a value of the Status column in the table, it's actually a separate boolean flag in the alert document. This means that the user cannot filter by `flapping` using the `Status` control, which is a bit counterintuitive, especially if we call it a state in the docs. See the last point of https://github.com/elastic/kibana/issues/183615#issue-2300060639

Reply:
Thanks, I've removed it from the list of status values but left the note about how to enable flapping detection. When it's possible to filter on that field, we can add more details.